native/android/app/build.gradle

```diff
@@ -162,9 +162,9 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: "libs", include: ["*.jar"])
-    compile "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
-    compile "com.facebook.react:react-native:+" // From node_modules
+    implementation fileTree(dir: "libs", include: ["*.jar"])
```

Comment:
Whats the deal here? can you link some docs? :)

Reply:
done.

Reply:
Where are the docs? :)

Reply:
I have linked to them in the comment. Will mention on the PR merge notes also.

Reply:
Gotcha, comment being [this one](https://github.com/Skyscanner/backpack/pull/850#issuecomment-398094488)

The pertinent bit:
> The `compile` configuration still exists but should not be used as it will not offer the guarantees that the api and `implementation` configurations provide.